packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeBlockerRoute } from '@deriv/shared';
+import LandscapeBlockerSvg from 'Assets/SvgComponents/settings/landscape-blocker.svg';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeBlockerRoute(pathname);
+
+    if (is_hidden_landscape_blocker) return null;
+
+    return (
+        <div id='landscape_blocker' className='landscape-blocker'>
+            <div className='landscape-blocker__icon'>
+                <LandscapeBlockerSvg />
+            </div>
+            <div className='landscape-blocker__message--landscape'>
+                Please adjust your screen size for optimal viewing.
+            </div>
+            <div className='landscape-blocker__message--portrait'>
+                Please adjust your <br />
```

Comment:
Will it be a good idea to remove `<br/>`? Please, ignore if irrelevant

Reply:
@kate-deriv I'm not sure that we really need to modify the landscape blocker, because I just copied it from index.html, and it will be deleted really soon when all packages modified to allow tablet view. I can modify it if you think we really need it